paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -336,7 +334,18 @@ def check_expected_slaves(
                         error_message + "But this is an sfr in cancelled_running state, so continuing anyways",
                     )
                     return None
-                error_message += "(refusing to go past %.2f%% missing instances)" % MISSING_SLAVE_PANIC_THRESHOLD
+                elif self.utilization_error < 0:
+                    log.warn(
+                        error_message + "Continuing because we are scaling down and we don't want the new "
```

Comment:
Just nitpicking on the error message but maybe:
"Continuing because we are scaling down and so we don't need any unregistered instances"

Reply:
Oh, yep that's better, will change.